openapi.yaml

```diff
@@ -15794,7 +15791,6 @@ components:
           type: integer
           description: >
             This Node Pool's unique ID.
-          x-linode-filterable: true
```

Comment:
@adammohammed I'm seeing that the node pool ID is filterable in the API--should we revert this line?

Reply:
Yep, this was a mistake good catch. Gonna double check the others too